hercules-ci-agent/hercules-ci-agent/Hercules/Agent/Client.hs

```diff
@@ -41,3 +45,7 @@ buildClient = useApi agentBuild $ Hercules.Agent.Client.client
 
 agentsClient :: Hercules.API.Agents.AgentsAPI ClientAuth (AsClientT ClientM)
 agentsClient = useApi agents $ Hercules.Agent.Client.client
+
+logsClient :: LogsAPI () (AsClientT ClientM)
```

Comment:
I'm a bit surprised this work, shouldn't the unit be auth type?

Reply:
It's not touched by the client. All it needs to know is the raw token. It's almost the same as `ClientAuth` which is `Auth '[JWT, Cookie] ()` except that for this API we know exactly how we want to authenticate, so `Auth` is fixed, but we do leave the signed contents of the token unspecified for the client.

Reply:
In fact we know for the all agent endpoints that it's going to be JWT, so we could take the same approach for those and specific in the API that it's going to be JWT only instead of whatever-`auth`